tensorboard/webapp/metrics/effects/metrics_effects_test.ts

```diff
@@ -89,8 +89,52 @@ describe('metrics effects', () => {
       selectors.getMetricsTooltipSort,
       TooltipSort.ALPHABETICAL
     );
+
+    overrideTagMetadata();
+    overrideRunToEid();
   });
 
+  function overrideTagMetadata() {
+    store.overrideSelector(selectors.getMetricsTagMetadata, {
+      scalars: {
+        tagDescriptions: {} as any,
+        tagToRuns: {
+          tagA: ['run1'],
+          tagB: ['run2', 'run3'],
+          tagC: ['run4', 'run5'],
+          tagD: ['run6'],
+        },
+      },
+      histograms: {
+        tagDescriptions: {} as any,
+        tagToRuns: {
+          tagA: ['run1'],
```

Comment:
Is it valid for there to be duplicate tags across scalars/histograms? Doesn't seem to make sense to me.

Reply:
Is there anything that prohibits this? Tags can appear in multiple experiments and they could have different data being logged.

Reply:
Ah ya, fair enough. That makes sense.